content/overview/pricing/rates.md

```diff
@@ -33,31 +33,31 @@ You can email [cloud-gov-inquiries@gsa.gov](mailto:cloud-gov-inquiries@gsa.gov)
 The [prototyping package]({{< relref "overview/pricing/prototyping.md" >}}) lets many teams build and deploy apps, with full access control. Prototyping packages cannot host production data; all apps are limited to the `*.app.cloud.gov` domain. Whole agencies or departments often purchase a single prototyping package.
 
 **Annual fee:** $15K<br>
-**Resource usage quota:** ~$99 per GB per month[*](#quota-pricing), up to any limit set by user<br>
+**Resource usage quota:** ~$100 per GB per month[*](#quota-pricing), up to any limit set by user<br>
 **Managed services:** Full access
 
 ### Open data
 
 Host one public-facing system with no confidentiality risk assessed. Open data packages include all the spaces you need and DNS support. This is the lightest production-level package.
 
 **Annual fee:** $10K<br>
-**Resource usage quota:** ~$99 per GB per month[*](#quota-pricing), up to 2GB<br>
+**Resource usage quota:** ~$100 per GB per month[*](#quota-pricing), up to 2GB<br>
 **Managed services:** Full access
 
 ### FISMA Low
```

Comment:
While we're editing this page, is there something we can add for indicating that these include dev, staging, production (or whatever configuration you desire)? I feel like we get that question often enough.